ci/install-upstream-wheels.sh

```diff
@@ -35,15 +36,16 @@ python -m pip install \
     --upgrade \
     git+https://github.com/dask/dask \
     git+https://github.com/dask/distributed \
-    git+https://github.com/zarr-developers/zarr \
-    git+https://github.com/Unidata/cftime \
-    git+https://github.com/rasterio/rasterio \
-    git+https://github.com/pypa/packaging \
+    git+https://github.com/h5netcdf/h5netcdf
     git+https://github.com/hgrecco/pint \
+    git+https://github.com/numba/numba \
```

Comment:
does `numba` require compilation? If so, it might be better to use pre-built packages (if `numba` provides those?).